File: Q2/Answer in Python/answer.py
number = int(input("Enter a number:"))
cntr = 0
i = 2
while True:
    if cntr >= number:break
    else:
        for j in range(2,i,1):
            if i % j == 0:
                break
        else:
            print(i,end = " ")
            cntr+=1
    i+=1

File: Q3/Answer in PHP/answer.php
<?php
$number = 32;
$arr = array();
$c = false;
for($i = 3,$cntr = 0;$i < $number;$i++){
		for($j = 2;$j < $i;$j++){
			if($i % $j == 0){
				$c = false;
				break;
			}
			else
				$c = true;
		}
		if ($c == true){
			$arr[$cntr] = $i;
			$cntr++;
		}
		$c = false;
}
$max = 0;
foreach($arr as $item){
	if($max < $item) $max = $item;
}
echo $max;
?>